src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -11,7 +11,10 @@
 			</div>
 		</TransitionWrapper>
 
-		<div v-if="!isSidebar" class="bottom-bar">
+		<div v-if="!isSidebar"
+			class="bottom-bar"
```

Comment:
That's quite a small area to catch:
![image](https://github.com/nextcloud/spreed/assets/93392545/40e99e3e-48c2-4f73-86e4-5a898c818024)

I thought about freezing the whole promoted VideoVue on hover. It already has a logic for that (and styles, but they're disable for `isBig`, and I'd revert it to provide feedback)

Reply:
That would be a large area from the screen and probability that the mouse is just there is high

Reply:
So fine? If user sees that it's hovered
Or we add some complex logic with timeout, like video players have

Reply:
> So fine? If user sees that it's hovered

That means, it could be easily hovered randomly (user left the mouse there) and now the actual speaker doesn't show up anymore.